packages/wrangler/src/index.tsx

```diff
@@ -937,6 +937,29 @@ export async function main(argv: string[]): Promise<void> {
 
       // TODO: if worker_dev = false and no routes, then error (only for dev)
 
+      function collectAllRoutes() {
+        const allRoutes: string[] = [];
```

Comment:
I don't think we'll want to collect all routes, only the ones that 'win', either args.routes if it's defined, or config.routes, or config.route